confirmer/confirmer.go

```diff
@@ -100,27 +100,40 @@ func (confirmer *Confirmer) checkPendingTxs(parent context.Context) {
 
 		if confirmed {
 			confirmer.logger.Infof("tx=%v has reached sufficient confirmations", tx.Hash.String())
-			confirmer.confirm(tx)
+			confirmer.confirm(ctx, tx)
 		}
 	})
 }
 
 // confirm sends the transaction to the dispatcher and marks it as confirmed.
```

Comment:
Can we update this comment to say something like "confirm sends the transaction to the dispatcher and marks it as confirmed if it receives a non-error response from the Darknodes."